### ISSUE TITLE
Question: how can I simply run FDnet via python code

### ISSUE THREAD

**Issue by noreplacementfordisplacement** [OrigamiSL/FDNet2023#3]
Hi all, I want to just run the exchange dataset without the .sh scripts. How do I best do this via a python script?

-Daniel
[Exchange.csv](https://github.com/OrigamiSL/FDNet-KBS-2023/files/12384271/Exchange.csv)

**Comment by OrigamiSL** [comment]
You can just run the python scripts in the shell script individually. For instance:
python -u main.py --model FDNet --data Exchange --root_path ./data/Exchange/ --features S  --label_len 96  --pred_len 96 --pyramid 0 --attn_nums 1 --kernel 3 --learning_rate 0.0001 --dropout 0.1 --criterion Standard --d_model 8 --timebed None --target Singapore --batch_size 16 --itr 10 --train
Ensure that you are using the correct environment before running the script.

**Comment by noreplacementfordisplacement** [comment]
Thanks! But I really want to run it in my pycharm environment (such that I can also look at the variables etc.). How can I run main.py with the above mentioned setting from my python ide pycharm? Thanks for the fast response!

**Comment by OrigamiSL** [comment]
Pycharm is also a convenient tool for me when operating in windows. Indeed, you can open a terminal window (hotkey is “Alt + F12”) inside the pycharm and then run the scripts. If you insist in only using the 'Run' button, you may change the default values of the variables in the config (args), which are described in main.py, to the desired ones for experiments on Exchange dataset.

**Comment by noreplacementfordisplacement** [comment]
For reference (below script helped to solve!)

python -u main.py --model FDNet --data Exchange --root_path ./data/Exchange/ --features S --label_len 96 --pred_len 96 --pyramid 0 --attn_nums 1 --kernel 3 --learning_rate 0.0001 --dropout 0.1 --criterion Standard --d_model 8 --timebed None --target Singapore --batch_size 16 --itr 10 --train
